modules/http/src/main/java/com/spotify/ffwd/http/HttpDecoder.java

```diff
@@ -61,7 +64,7 @@ protected void decode(ChannelHandlerContext ctx, FullHttpRequest in, List<Object
                 }
 
                 throw new HttpException(HttpResponseStatus.METHOD_NOT_ALLOWED);
-            case "/v1/batch":
+            case "/v1/batch": case "/v2/batch":
```

Comment:
does this syntax just mean in either case do the following? I thought the v1/batch endpoint was removed above 🤔  `private static final String V1_BATCH_ENDPOINT = "v1/batch";`

Reply:
Yes it does, Regarding  "v1/batch",   ffwd-http-client will continue to send metric using that uri => https://github.com/spotify/ffwd-http-client/blob/master/src/main/java/com/spotify/ffwd/http/RawHttpClient.java#L39

Reply:
👍  thanks for clarifying